ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/individual-journey/individual-journey.factory.spec.ts

```diff
@@ -3,26 +3,34 @@ import { IndividualJourneyFactory } from './individual-journey.factory';
 import { IndividualJourney } from './individual-journey';
 import { JourneyRoutingListenerService } from '../base-journey/services/journey-routing-listener.service';
 import { JourneyStepComponentBindings } from './services/journey-component-bindings';
+import {IndividualJourneyService} from './services/individual-journey.service';
+import {MutableIndividualSuitabilityModel} from './mutable-individual-suitability.model';
 
 describe('IndividualJourneyFactory', () => {
     let suitabilityService: jasmine.SpyObj<SuitabilityService>;
     let routingListener: jasmine.SpyObj<JourneyRoutingListenerService>;
+    let individualJourneyService: jasmine.SpyObj<IndividualJourneyService>;
     let journey: jasmine.SpyObj<IndividualJourney>;
     let factory: IndividualJourneyFactory;
     const bindings = new JourneyStepComponentBindings();
 
     beforeEach(() => {
-        suitabilityService = jasmine.createSpyObj<SuitabilityService>(['getAllSuitabilityAnswers']);
-        suitabilityService.getAllSuitabilityAnswers.and.returnValue(Promise.resolve([]));
-        routingListener = jasmine.createSpyObj<JourneyRoutingListenerService>(['initialise']);
-        journey = jasmine.createSpyObj<IndividualJourney>(['forSuitabilityAnswers']);
-        factory = new IndividualJourneyFactory(journey, suitabilityService, bindings, routingListener);
+      suitabilityService = jasmine.createSpyObj<SuitabilityService>(['getAllSuitabilityAnswers']);
+      suitabilityService.getAllSuitabilityAnswers.and.returnValue(Promise.resolve([]));
+      routingListener = jasmine.createSpyObj<JourneyRoutingListenerService>(['initialise']);
+      journey = jasmine.createSpyObj<IndividualJourney>('journey', ['forSuitabilityAnswers', 'redirect']);
```

Comment:
no need to supply 'journey' as the type you're mocking is provided by `<IndividualJourney>`:

```typescript
journey = jasmine.createSpyObj<IndividualJourney>(['forSuitabilityAnswers', 'redirect']);
journey.redirect.subscribe = () => {};
individualJourneyService = jasmine.createSpyObj<IndividualJourneyService>(['get', 'set']);
```